docs/production/ingesters-with-wal.md

```diff
@@ -74,3 +74,14 @@ You can run it as a Kubernetes job which will:
 3. And flush all the chunks.
 
 This job is to be run for all the PVCs linked to the ingesters that you missed hitting the shutdown endpoint as a first option.
+
+## Additional notes
+
+* If you have lots of ingestion with the WAL replay taking a longer time, you can try reducing the checkpoint duration to `10-15m`. This would require slightly higher disk bandwidth for writes (still less in absolute terms), but it will reduce the WAL replay time overall.
```

Comment:
> reducing the checkpoint duration

I would mention the CLI flag to do it. Also I would keep it simple a give a value to set (not a range), so I would pick either `10m` or `15m`.

Reply:
I will mention a specific value. And about CLI flag, I will add that too. As the flags were mentioned at the start of the doc, I felt it was not necessary.